src/styles/styles.scss

```diff
@@ -1,4 +1,48 @@
 @import 'theme';
+@import 'forms';
+
+html {
+  height: 100%;
+  box-sizing: border-box;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
 body{
+  position: relative;
   margin:0px;
+  padding-bottom: 6rem;
+  min-height: 100%;
+}
+
+app-footer{
```

Comment:
Footer component scss file?

Reply:
If I move app-footer{ into the footer.scss file, it doesn't take effect.  I think the footer.scss file is for formatting items within the footer itself and not formatting the footer.